oracle-linux-image-tools/cloud/vagrant-virtualbox/provision.sh

```diff
@@ -134,7 +44,13 @@ cloud::install_agent()
 {
   echo_message "Install guest agent"
   local additions="/mnt/VBoxLinuxAdditions.run"
-  yum install -y ${YUM_VERBOSE} make gcc bzip2 kernel-uek-devel
+  yum install -y ${YUM_VERBOSE} make gcc bzip2
+
+  if [[ "${KERNEL,,}" = "uek" ]]; then
```

Comment:
Is there a chance that someone could specify `uek` but reboot the VM into the RHCK (or vice-versa)? If so, we probably need both `kernel-uek-devel` and `kernel-devel` installed so that the Guest Additions can be rebuilt regardless of which kernel is booted.

Reply:
No, there is only one kernel installed, so you can't reboot with the _wrong_ one.
Switching would mean installing a new kernel and manually updating the default boot...

Reply:
So why check for a kernel at all?

Reply:
Because you need to know which one!